doc/source/getting_started/docker/linux_container.rst

```diff
@@ -0,0 +1,178 @@
+.. _ref_linux_docker:
+
+Geometry Service as a Linux Docker container
```

Comment:
The links are on purpose, especially when dealing with large documents I find it useful to have the TOC at the beginning. The "problem" is that it also shows up on the right side - but that's also good because as you scroll down, you still have access to the TOC. @PipKat any feedback here?

Reply:
I just found out one can remove the links of the headers and keep the TOC. Going ahead with that.